### ISSUE TITLE
FiraGO Serif?

### ISSUE THREAD

**Issue by inferno986return** [bBoxType/FiraGO#15]
I know I am only one user, but I would like to make a request for FiraGO Serif. It would be useful to have a complete superfamily.

**Comment by carrois** [comment]
That’s true. You are not the only one ;-) We are also keen on making a Serif. But this is a huge task, we will definitely need funding here.
